app/views/provider/admin/cms/templates/_quick_links.html.erb

```diff
@@ -14,8 +14,8 @@
     <li>
       <p>
         <%= icon('question-circle fa-li') %>
-        Follow the
-        <%= link_to 'Quickstart guide', I18n.t('docs.hero_api_portal', docs_base_url: docs_base_url) %>
+        Creating the
+        <%= link_to 'Developer Portal', I18n.t('docs.hero_api_portal', docs_base_url: docs_base_url) %>
```

Comment:
```suggestion
        <%= link_to 'Creating the Developer Portal', I18n.t('docs.hero_api_portal', docs_base_url: docs_base_url) %>
```

Reply:
So `Creating the Creating the Developer Portal`?

Reply:
Suggestion: 

Something like: `Follow the documentation on <link>Creating the Developer Portal</link>`